config/prometheus/monitor.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   labels:
-    control-plane: controller-manager
+    app.kubernetes.io/managed-by: ramen
```

Comment:
Can you explain this change? The commit message says that we add a new lable. The actual change is removing existing label (why?) and adding a new one (why?).

Reply:
i have updated the commit message and the changes are different now. basically, few of the resources did not have label like `app: ramen-hub`. fixing this was essential because resources like ServiceMonitor was to open to  alerts   when other pods were installed.